pkgs/applications/misc/keepassx/community.nix

```diff
@@ -43,13 +43,13 @@ with lib;
 
```

Comment:
It would be nice if you could drop the unused buildInputs that the bot found.

Reply:
yes, just fixing those up.